src/lib.rs

```diff
@@ -211,6 +217,20 @@ impl Model {
         self.set_row_upper(row, value);
         self.set_row_lower(row, value);
     }
+
+
+    /// Add a special ordered set constraint, preventing all but one variable
+    /// in a set from being non-zero at the same time.
```

Comment:
@TeXitoi : would that be okay ?  I added a short explanation and a link : 

```suggestion
    /// in a set from being non-zero at the same time.
    /// weights can be used as hints to the optimizer to improve the resolution speed.
    /// In case you don't have any weights for your variables, you can use 1, 2, 3, ... 
    /// For more information about SOS weights, see: http://lpsolve.sourceforge.net/5.5/SOS.htm  
```

Reply:
From your link:

> VarWeight determines the adjacency of the variables. This is an importand piece of information. Remember that the SOS definition defines that the non-zero variables must be adjacent. The VarWeight values define this adjencency.

Reply:
There is no "adjacency" for type 1 SOS. Would  you like me to copy-and paste the quote above in the docs for sos2 ?

Reply:
Yes please